godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/pool/PoolByteArray.kt

```diff
@@ -147,28 +147,28 @@ class PoolByteArray : NativeCoreType<godot_pool_byte_array>, Iterable<UByte> {
      * Not available in the Gdnative API and no workaround for now
      */
     private fun compress(compressionMode: Int = 0): PoolByteArray {
-        TODO()
+        return PoolByteArray()
     }
```

Comment:
I would not just return a empty array. I would let it crash and state so in the kDoc with `@throws` otherwise this method just "works" and the user might search for the bug for a long time until he realizes that this method in reality does not "work"

Reply:
Right, I'll do that.
I also considered removing them but I want them around to notify users that we didn't just forget those methods, just that we can't implement them.

Reply:
What about `GDNativeNotDefinedException` ? :)

Reply:
[It exists already, but I can inherit from it](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-not-implemented-error/)